tests/018_numeric_progress.test

```diff
@@ -30,8 +30,14 @@ EOF
 
 $FWUP_CREATE -c -f $CONFIG -o $FWFILE
 
+echo $ACTUAL_OUTPUT_FILE
```

Comment:
Yes, I was trying to figure out what was going on because in the case of output redirection, it needs to be in Linux format instead of Windows because apparently it's not Wine that redirects, but the shell that Wine is running the executable in (which is Bash). Fun stuff.